script/foreman-debug

```diff
@@ -86,21 +86,24 @@ add_file() {
   MIME=$(file -bi "$FILE" | cut -d\; -f1)
   case $MIME in
         application/x-gzip)
-          zcat "$FILE" | sed -r "$FILTER" > "$DIR$FILE.txt"
-	  touch -c -r "$FILE" "$DIR$FILE.txt"
+          OUTFILE=$(echo "$FILE" | sed s/.gz$//)
+          zcat "$FILE" | sed -r "$FILTER" > "$DIR$OUTFILE"
+          touch -c -r "$FILE" "$DIR$OUTFILE"
```

Comment:
@lzap Tried to merge `$DIR$FILE` into a single var, please see if it looks okay.